### DISCUSSION TITLE
Pass quotes correctly from frontmatter

### DISCUSSION THREAD

**Discussion by Obspogon** [11ty/eleventy#2631] [Q&A] [answered]
If I pass a frontmatter value like this:
```
customStyle: 'body { background-image: url("/pics/image.png"); '
```

The quotes get replaced in the HTML:
```
body { background: url(&#39;/pics/image.png&#39;); }
```

How do I pass quotes safely through frontmatter

**Comment by TigersWay** [comment]
A little bit more information would help 😇 Nunjucks/Liquid/Markdown/Combined? Layout? etc?
As usual a visible light repro would be a good start.

**Reply by Obspogon** [reply]
nunjucks

**Comment by pdehaan** [comment, marked as answer]
@Obspogon Assuming you're using Nunjucks, you'll need to use the [`safe` filter](https://mozilla.github.io/nunjucks/templating.html#safe) to avoid auto-escaping.

**Reply by Obspogon** [reply]
How do I use the safe filter in top-level frontmatter? i.e.:

```
---
layout: base
customStyle: 'body {background-image: url("/pics/img.png");}' | safe // dosen't work
---
```

**Reply by pdehaan** [reply]
Close, but you would use the filter in your Nunjucks layout (or however you're rendering the data).
So something like `{{ customStyle | safe }}`, depending on where/how you're using the front matter value.

**Reply by Obspogon** [reply]
oh
